src/bootstrap/dist.rs

```diff
@@ -897,12 +898,12 @@ impl Step for PlainSourceTarball {
 
         // Create the version file
         builder.create(&plain_dst_src.join("version"), &builder.rust_version());
-        if let Some(sha) = builder.rust_sha() {
-            builder.create(&plain_dst_src.join("git-commit-hash"), &sha);
```

Comment:
Yeah, I overlooked this edit. I think we'll want to keep the git-commit-hash file around; it's consumed by build-manifest (https://github.com/rust-lang/rust/blob/744e397d8855f7da87d70aa8d0bd9e0f5f0b51a1/src/tools/build-manifest/src/versions.rs#L144).

We can probably also fix that code, but I think it's best to leave the file in place.

Reply:
Oh, my bad, I didn't realize other parts of code used it. Should I open a PR to add that back?

Reply:
If you could open a PR that would be great :)

Reply:
rust-lang/rust#102610 is now open :)